packages/mesos-modules/build

```diff
@@ -1,15 +1,53 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
+# Update the image and download build tools.
+apt-get update -qq
+apt-get install -y autoconf libtool make pkg-config
```

Comment:
The dependency chain for `mesos-modules` is:
```
 mesos-modules
      |
    mesos
/      |     \
|  libevent   curl
\      |      / \
 --- openssl   python-requests
                     |
                  python
```
`openssl` and `python` in particular are dependencies for tons of other packages, so instead of selectively adding a new build image, it seems better to change the default image (the second option you referred to).